2 commits, an initial translation of c00/30/04->14 and an extra adding sounds as per indication. planning to use dupe tool after this (#63)
IllusionMods/HS2-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/30/05/2/translation.txt

//
// Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
//

> これは……お見苦しいものをお見せして申し訳ございません。=Oh my... I'm sorry to have shown you something so unsightly.

Reviewer: ```suggestion
これは……お見苦しいものをお見せして申し訳ございません。=Oh my... I'm sorry you had to see this.
```
